src/pages/components/button/usage.mdx

```diff
@@ -542,107 +539,805 @@ recommended action labels.
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not place the icon to the left of the text on the button.">
+
+![Do not place the icon to the left of the text on the button.](images/button_usage_button_with_icon_2.png)
+
+  </DoDont>
+</DoDontRow>
+
+<DoDontRow>
+  <DoDont type="do" caption="Do use 8px spacing between the label and the icon in a ghost button.">
+
+![Do use 8px spacing between the label and the icon in a ghost button.](images/button_usage_button_with_icon_3.png)
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not leave too much space between the label and the icon in a ghost button.">
+
+![Do not leave too much space between the label and the icon in a ghost button.](images/button_usage_button_with_icon_4.png)
+
+  </DoDont>
+</DoDontRow>
+
+#### Universal actions with well-established icons
+
+When deciding whether to add an icon to a button, it’s important to consider how
+recognizable the icon is and whether, in other contexts, a similar icon may be
+used for a different meaning. We want to avoid the same icon being used for
+completely different actions.
+
+For this reason, we have defined a set of universal actions suitable for adding
+an icon alongside the button label. These have clearly defined and widely
+recognized icons within IBM Software and beyond.
+
+| Name of action | Associated icon                                    | Name in icon library |
+| -------------- | -------------------------------------------------- | -------------------- |
+| Create/Add     | <Add size={16} aria-label="Add" />                 | Add                  |
+| Edit           | <Edit size={16} aria-label="Edit"/>                | Edit                 |
+| Copy           | <Copy size={16} aria-label="Copy"/>                | Copy                 |
+| Delete         | <TrashCan size={16} aria-label="Trash can"/>       | Trash can            |
+| Remove         | <SubtractAlt size={16} aria-label="Subtract alt"/> | Subtract alt         |
+| Export         | <Export size={16} aria-label="Export"/>            | Export               |
+| Upload         | <Upload size={16} aria-label="Upload"/>            | Upload               |
+| Download       | <Download size={16} aria-label="Download"/>        | Download             |
+| Play/Start     | <Play size={16} aria-label="Play"/>                | Play                 |
+| Pause          | <Pause size={16} aria-label="Pause"/>              | Pause                |
+| Stop           | <StopOutline size={16} aria-label="Stop outline"/> | Stop outline         |
+| Refresh        | <Restart size={16} aria-label="Restart"/>          | Restart              |
+
+Do you think a universal action with a clearly defined icon is missing? Let us
+know [here](https://github.com/carbon-design-system/carbon/issues/new/choose).
+
+Icons that are not in this list can be used in buttons, as long as the icon
```

Comment:
I think we should move the sentence up with the other content above the table. These three paragraphs just feel very choppy here. 
<img width="857" alt="image" src="https://github.com/user-attachments/assets/0ec3cea2-32bb-46b1-b208-ce6ea93e6d92" />